### PR TITLE
Added files for compression benchmark

### PR DESCRIPTION
**Pull request by KarthikL1729**
Added all the files for the compression benchmark. Tested with vSwarm-proto and docker-compose.

### CODE REVIEW

File: benchmarks/compression/python/server.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import os
+import zlib
+import tracing
+
+LAMBDA = os.environ.get('IS_LAMBDA', 'no').lower() in ['true', 'yes', '1']
```
[the remark this answers is not in the document]
Comment (KarthikL1729) [reply benchmarks/compression/python/server.py:5]: Shall I remove all the lambda related blocks?

File: benchmarks/compression/yamls/knative/kn-compression-python.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          ports:
+            - name: h2c
+              containerPort: 50000
+          args:
+            - --addr=0.0.0.0:50000
+            - --function-endpoint-url=0.0.0.0
+            - --function-endpoint-port=50051
+            - --function-name=compression-python
+            - --value=metamorphosis.txt
+            - --profile-function=true
```
[the remark this answers is not in the document]
Comment (KarthikL1729) [reply benchmarks/compression/yamls/knative/kn-compression-python.yaml:42]: Will remove it.

**Review by dhschall — CHANGES_REQUESTED** [review]
Looks all quite good. Just a minor change to make the test work

File: benchmarks/compression/yamls/docker-compose/dc-compression-python.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  relay:
+    image: vhiveease/relay-latency:latest
+    entrypoint:
+      - /app/server
+      - --addr=0.0.0.0:50000
+      - --function-endpoint-url=compression-python
+      - --function-endpoint-port=50051
+      - --function-name=compression-python
+      - --value=metamorphosis.txt
+    environment:
```
[the remark this answers is not in the document]
Comment (KarthikL1729) [reply benchmarks/compression/yamls/docker-compose/dc-compression-python.yaml:45]: Oh I thought you meant the profile function flag alone, will remove all.

[the remark this answers is not in the document]
Comment (KarthikL1729) [reply benchmarks/compression/yamls/docker-compose/dc-compression-python.yaml:45]: I think removing the value flag broke it before though

[the remark this answers is not in the document]
Comment (KarthikL1729) [reply benchmarks/compression/yamls/docker-compose/dc-compression-python.yaml:45]: It does break the CI tests now as well, will leave it in.

File: .github/workflows/e2e-compression.yml [hunk 4]
```diff
@@ -104,7 +104,7 @@ jobs:
 
       - name: invoke the chain
         run: |
-          ./tools/bin/grpcurl -plaintext -d '{"regenerate": "false"}' localhost:50051 compression.TODO
+          ./tools/bin/grpcurl -plaintext -d '{"regenerate": "false"}' localhost:50050 compression.TODO
```
[the remark this answers is not in the document]
Comment (KarthikL1729) [reply .github/workflows/e2e-compression.yml]: this seems to break the docker compose workflow, do I reset it?

File: .github/workflows/e2e-compression.yml [hunk 5]
```diff
@@ -104,7 +104,7 @@ jobs:
 
       - name: invoke the chain
         run: |
-          ./tools/bin/grpcurl -plaintext -d '{"regenerate": "false"}' localhost:50051 compression.TODO
+          ./tools/bin/grpcurl -plaintext -d '{"regenerate": "false"}' localhost:50000 compression-python.TODO
```
Comment (dhschall) [inline .github/workflows/e2e-compression.yml:107]:
```suggestion
         ./tools/bin/grpcurl -plaintext localhost:50000 helloworld.Greeter.SayHello
```

**Review by dhschall — APPROVED** [review]
All good! 
Thank you @KarthikL1729 !

**Pull request merged**